src/ert/scheduler/openpbs_driver.py

```diff
@@ -36,13 +36,14 @@
 QDEL_JOB_HAS_FINISHED: int = 35
 QDEL_REQUEST_INVALID: int = 168
 
-QSUB_EXIT_CODES = [
+QSUB_EXIT_CODES_TO_RETRY = [
     QSUB_INVALID_CREDENTIAL,
     QSUB_PREMATURE_END_OF_MESSAGE,
     QSUB_CONNECTION_REFUSED,
 ]
 
-QDEL_EXIT_CODES = [QDEL_REQUEST_INVALID, QDEL_JOB_HAS_FINISHED]
+QDEL_EXIT_CODES_TO_RETRY = [QDEL_REQUEST_INVALID]
```

Comment:
Nitpick: the rest of the Scheduler code uses "returncode" (from `asyncio.process.Process`'s `returncode` property), not "exit_code".

Also, perhaps this should be inlined with the `qdel` code?

Reply:
I've removed the groups and inject them directly as individual return_codes :+1: